.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,28 +4,25 @@ about: Request a new feature
 
 ---
 
-<!--
-DELETE ME
+# Checklist
+<!-- Place an x in the boxes to tick them: [x] -->
 
-Make sure you are using the latest version of AntennaPod. Perhaps the feature you are looking for has already been implemented.
+- [ ] I have used the search function to see if someone else has already submitted the same feature request.
+- [ ] I will only create one feature request per issue.
```

Comment:
"Please do/I will not combine multiple feature requests in one issue"
or:
"Please/I will create an issue for each individual feature request"